### PR TITLE
Update exercise check_http

### PR DESCRIPTION
**Pull request by mouselike**
- Added a missing point for configuring the advanced service.
- Corrected the numbered list.

### CODE REVIEW

**Comment by Cat5TV** [comment]
Thanks! Good catch.

Can you please remove the space character you'd added on line 79 to fix the automatic bullets, and then revert 9, 10 with #, #?

Cheers!

File: gettingstarted/exercises/check_http.rst [hunk 1]
```diff
@@ -71,20 +71,21 @@ Steps
...
   #. Name your service: *Web Site (SSL)*
   #. Give it an alias: *Uptime of SSL Web Site*
-  #. Set the check period and notification period to 24x7
+  #. Set the check command to *check_http*
+  #. Set the check period and notification period to *24x7*
   #. In *assign advanced-service to hostgroup*, highlight the hostgroup we created (web_site_ssl) and press the green arrow to add it to the selected items list.
   #. Under *contact groups* be sure to add *admins* as well. Otherwise, you won’t receive notifications.
   #. Set your notifications as follows:
-
+
```
Comment (Cat5TV) [inline gettingstarted/exercises/check_http.rst]: Guessing this extraneous space character is what broke the automatic bullet numbering on you. Please test that theory and if yes, revert the static 9, 10 bullets to #, #. Thanks!

File: gettingstarted/exercises/check_http.rst [hunk 2]
```diff
@@ -71,20 +71,21 @@ Steps
...
     - max check attempts: 10
     - check interval: 1
     - retry interval: 5
     - first notification delay: 10
     - notification interval: 30
     - notification options: w,u,c,r,f
-
-  #. Finally, set your service parameters to: `-S -\\-sni`
-  #. Save your advanced service.
+
```
Comment (Cat5TV) [inline gettingstarted/exercises/check_http.rst]: These spaces will also break automatic bullets numbering. Please remove these.

File: gettingstarted/exercises/check_http.rst (hunk 1 above)
Comment (mouselike) [reply gettingstarted/exercises/check_http.rst]: I replaced the # with static numbers because I think the unordered list breaks the automatic bullet numbering. Further down on the same page the same thing happens at point 5.

**Comment by mouselike** [comment]
I think I fixed the issues at both places. Please take a look.

File: gettingstarted/exercises/check_http.rst [hunk 3]
```diff
@@ -71,18 +71,19 @@ Steps
...
-    - max check attempts: 10
-    - check interval: 1
-    - retry interval: 5
-    - first notification delay: 10
-    - notification interval: 30
-    - notification options: w,u,c,r,f
-
+     
+     - max check attempts: 10
+     - check interval:
```
Comment (Cat5TV) [inline gettingstarted/exercises/check_http.rst]: Why did you remove the interval?

Comment (mouselike) [reply gettingstarted/exercises/check_http.rst]: Sorry, my mistake! 🙈 
Added it again.

Comment (Cat5TV) [reply gettingstarted/exercises/check_http.rst]: Thanks  :)

**Comment by Cat5TV** [comment]
Looks good, thank you. Will merge and watch readthedocs to see if there are any issues with the rendering (nested bullets can indeed be a pain sometimes)  :)

Thanks!
